src/App.js

```diff
@@ -1,10 +1,36 @@
 import "./App.css";
-import logo from "./logo.png";
+import { LeftNav } from "./Components/Navigation/leftNav";
+import { Post } from "./Components/Post/Post";
+import { Sidecard } from "./Components/sidecard/Sidecard";
+import { Bookmark } from "./Pages/bookmark/Bookmark";
+import { Liked } from "./Pages/liked/Liked";
```

Comment:
use index file  and export all this file from index file so your code looks more cleaner